### PR TITLE
Danila bladyko hm8 task3

### PR DESCRIPTION
**Pull request by TLittleProgrammer**

### CODE REVIEW

**Review by vladis1350 — DISMISSED** [review]
lgfm

File: homework/Bladyko Danila/src/by/jrr/bean/Palindrome.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        int size = 0;
+
+        for(int i = 0; i < s.length(); i++) {
+            if(s.charAt(i) >= 'a' && s.charAt(i) <= 'z') {
+                arrayOfChars[size++] = s.charAt(i);
+            }
+        }
+        int legnth = size / 2;
+        for(int i = 0; i < legnth; i++) {
+            if(arrayOfChars[i] != arrayOfChars[size - i - 1]) {
```
Comment (javaGuruBY) [inline homework/Bladyko Danila/src/by/jrr/bean/Palindrome.java]: looks complicated

File: homework/Bladyko Danila/src/by/jrr/test/PalindromeTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    public void testShouldAnswerTrue() throws Exception {
+        palindrome = new Palindrome();
+
+        boolean actualFirstResult = palindrome.isPalindrome("aassaa");
+        boolean actualSecondResult = palindrome.isPalindrome("!!a!a?s,s.a/[a}");
+
+        boolean expectedResult = true;
+
+        checkGoodAnswer(actualFirstResult, expectedResult, "Test 1");
```
Comment (javaGuruBY) [inline homework/Bladyko Danila/src/by/jrr/test/PalindromeTest.java]: test 1-2-3-4-5-6-7-? 
give meaningful names

**Review by javaGuruBY — CHANGES_REQUESTED** [review]
merge conflict should be resolved

**Review by vladis1350 — APPROVED** [review]
lgfm

**Review by javaGuruBY — APPROVED** [review]
lgfm

**Pull request merged**